.github/workflows/ci.yml

```diff
@@ -201,6 +201,38 @@ jobs:
           docker_registry: ${{ inputs.docker_registry }}
           docker_username: ${{ inputs.docker_username }}
           docker_password: ${{ secrets.docker_password }}
+          deploy_annotate_pr: ${{ inputs.deploy_annotate_pr }}
+          pull_request_number: ${{ inputs.pull_request_number }}
+      
+      # - name: Read data from file
+      #   if: matrix.check == 'compose:test'
+      #   id: read_data
+      #   run: |
+      #     ls
+      #     cov_sum=$(cat app/output_directory/coverage-summary.txt)
+      #     echo "::set-output name=cov_data::$cov_sum"
+      #     echo $cov_sum
+      #   shell: bash
+      
+      - name: Code Coverage Report
```

Comment:
Is it possible to move this step and next step from this to `.github/actions/check` instead?